components/popover/style.scss

```diff
@@ -70,7 +70,7 @@
 	width: calc( 100vw - 20px );
 
 	@include break-medium {
-		width: 300px;
+		width: 320px;
```

Comment:
Is this needed for all popovers or should apply only to the inserter?

Reply:
Great point. I moved the width stuff around to better spots. Had to also give a width to the publish dropdown now as it inherited teh width from here. 

![screen shot 2017-10-21 at 09 43 33](https://user-images.githubusercontent.com/1204802/31849546-7c4f8796-b644-11e7-913a-83f90947a497.png)